packages/css/src/image/README.md

```diff
@@ -0,0 +1,16 @@
+# Image
+
+Toont een afbeelding.
```

Comment:
Weren't we writing docs in English from now on?

Reply:
Well, yes, but maybe not before we translated the rest all at once to prevent having two languages live at the same time?

Reply:
I thought we wanted to do it incrementally? We could do it at once I guess, but that's quite a big job

Reply:
I’m happy to block some time for a big bang.